fenix/app/src/main/res/values/strings.xml

```diff
@@ -2235,7 +2235,7 @@
     <!-- Flag showing that the review quality check feature is work in progress. -->
     <string name="review_quality_check_beta_flag" tools:ignore="UnusedResources">Beta</string>
     <!-- Content description (not visible, for screen readers etc.) for opening browser menu button to open review quality check bottom sheet. -->
-    <string name="browser_menu_review_quality_check">Open Review quality check</string>
+    <string name="browser_menu_review_quality_check">Open review checker</string>
```

Comment:
https://mozilla-l10n.github.io/documentation/localization/making_string_changes.html

You need a new ID. Also, what about the next string?